wdl-lint/src/rules/blank_lines_between_elements.rs

```diff
@@ -557,6 +577,30 @@ fn check_prior_spacing(
     }
 }
 
+/// Check that the node's last token does not have a blank before it.
+fn check_last_token(
+    syntax: &SyntaxNode,
+    state: &mut Diagnostics,
+    exceptable_nodes: &Option<&'static [SyntaxKind]>,
+) {
+    let prev = syntax
+        .last_token()
+        .expect("Node should have last token")
+        .prev_sibling_or_token();
```

Comment:
Since we know we're looking for a whitespace token, we can cast the element into a token like so:

```suggestion
        .prev_sibling_or_token()
        .and_then(SyntaxElement::into_token);
```

This will enable us to call `text()` to get the token's text without having to allocate a string below.